app/src/main/sentences/fr/telephone.dslf

```diff
@@ -0,0 +1,4 @@
+telephone: high
+(r?appel<er?)|(sonne<r)|(téléphone<r|z?>(à?))|(compose<r|z?>)|(contacte<r|z?>)|(le? numéro de) .who.;
```

Comment:
In addition to Stypox's suggested changes, it can't match "Appelle .who." because an "l" is missing. 

```suggestion
(r? appel<le|er?>)|(sonne<r?>)|(téléphone<r|z?> (à?))|(compose<r|z?>)|(contacte<r|z?>)|(le? numéro de) .who.;
```